Libraries/CO.CDP.Organisation.WebApiClient.Tests/OrganisationClientIntegrationTest.cs

```diff
@@ -48,10 +48,15 @@ public async Task ItTalksToTheOrganisationApi()
             )
         };
         var roles = new List<PartyRole> { PartyRole.Supplier };
+        var buyerInfo = new BuyerInformation(
+            buyerType:"Buyer Type",
+            devolvedRegulations: [DevolvedRegulation.NorthernIreland, DevolvedRegulation.Wales]);
+
         var newOrganisation = new NewOrganisation(
             personId: unknownPersonId,
             additionalIdentifiers: additionalIdentifiers,
             addresses: [address],
+            buyerInfo: buyerInfo,
```

Comment:
We agreed to have a separate call to `PATCH /organisations/123/buyer-information` to keep buyer and supplier stuff separate from the core organisation details. We should not submit buyer info with the register organisation call.